src/main/java/com/rocketden/main/dto/problem/CreateTestCaseRequest.java

```diff
@@ -1,4 +1,12 @@
 package com.rocketden.main.dto.problem;
 
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
 public class CreateTestCaseRequest {
+    private String input;
+    private String output;
+    private boolean hidden = false;
```

Comment:
Also, thoughts on allowing the user to set the value of `hidden`?

Reply:
Yeah sounds good, I think I'll leave this for a later time since making enums can be a hassle and might cause the PR to be more bloated (there should be a card in the backlog for this though). 

Second point: yep I agree (it actually should already be a feature, the false is just the default value :))

Reply:
Ah gotcha, that makes sense -- thanks!